root/admin/EmailSearch.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * @flow
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import FormRowText from '../components/FormRowText';
+import FormSubmit from '../components/FormSubmit';
+import {withCatalystContext} from '../context';
+import Layout from '../layout';
+
+import UserList from './components/UserList';
+
+type Props = {
+  +$c: CatalystContextT,
+  +form: FormT<{
+    +email: ReadOnlyFieldT<string>,
+  }>,
+  +results?: $ReadOnlyArray<EditorT>,
+};
+
+const EmailSearch = ({
+  $c,
+  form,
+  results,
+}: Props) => (
+  <Layout fullWidth title={l('Search users by email')}>
+    <div id="content">
+      <h1>{l('Search users by email')}</h1>
+
+      <form action="/admin/email-search" method="post">
```

Comment:
We can't have CSRF tokens here because `/admin/email-search` is invoked via query parameters from the user profile page. (I don't think they're useful here anyway, because this form doesn't modify the database in any way.)